### DISCUSSION TITLE
Docker Desktop Windows / WSL2 - unable to browse Linux filesystem

### DISCUSSION THREAD

**Discussion by Dolphsps** [harness/drone-ci-docker-extension#64] [Q&A]
Hi.  Looking to leverage the drone extension on Docker Desktop (v4.15.0) on Windows 11 with Ubuntu running under WSL2. 

When asked to select the base directory to scan for pipline files, I am unable to select my WSL2 filesystem where my code and pipelines are stored.

Has anyone else managed to get this running?

**Comment by kameshsampath** [comment]
@Dolphsps  - I am sorry I have added Windows support yet into to it.
